test/langtools/jdk/javadoc/doclet/testOverview/TestOverview.java

```diff
@@ -62,17 +62,25 @@ public void test2() {
                     "-sourcepath", testSrc("msrc"),
                     "p1", "p2");
         checkExit(Exit.OK);
-        checkOverview();
+        checkOverview("acme/");
     }
 
-    void checkOverview() {
+    void checkOverview(String modulePrefix) {
         checkOutput("index.html", true,
                 """
                     <main role="main">
                     <div class="header">
                     <h1 class="title">Document Title</h1>
                     </div>
                     <div class="block">This is line1. This is line 2.</div>
-                    """);
+                    <dl class="notes">
+                    <dt>See Also:</dt>
+                    <dd>
+                    <ul class="see-list">
+                    <li><a href="%1$sp1/C.html" title="class in p1"><code>C</code></a></li>
```

Comment:
What's with the funky URL?

Reply:
That's just a format string specifier "%1$s", meaning the first format argument (see invocation of `formatted` method below to adapt the string to a module or non-module context). 

I realize now that I could have omitted the argument index part of the format specifier, will update the PR if that is true.

Reply:
Doh, silly me; of course.